tests/fsharp/typecheck/sigs/neg07.bsl

```diff
@@ -23,6 +23,13 @@ neg07.fs(36,11,36,27): typecheck error FS0026: This rule will never be matched
 
 neg07.fs(46,15,46,27): typecheck error FS0039: The record label 'RecordLabel1' is not defined. Maybe you want one of the following:
    R.RecordLabel1
+   R.RecordLabel2
+
+neg07.fs(46,33,46,45): typecheck error FS0039: The record label 'RecordLabel2' is not defined. Maybe you want one of the following:
+neg07.fs(47,17,47,55): typecheck error FS0025: Incomplete pattern matches on this expression.
+neg07.fs(47,37,47,49): typecheck error FS0039: The record label 'RecordLabel2' is not defined. Maybe you want one of the following:
```

Comment:
The empty suggestion list looks strange...

Reply:
Hm, yes, indeed. It seems the testing framework use here doesn't check the order of the strings. I've reordered the baseline.